### PR TITLE
ECOM-7199 Remove program_id field

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
@edx/ecommerce

### CODE REVIEW

File: credentials/apps/credentials/models.py [hunk 1]
```diff
@@ -245,8 +245,6 @@ class ProgramCertificate(AbstractCertificate):
     """
 
     program_uuid = models.UUIDField(db_index=True, unique=True, null=False, blank=False, verbose_name=_('Program UUID'))
```
Comment (davec-edx) [inline credentials/apps/credentials/models.py]: I realize this is a previous commit...but there should be help text for the UUID.

**Comment by davec-edx** [comment]
other than my comment, LGTM

File: credentials/apps/credentials/models.py (hunk 1 above)
Comment (MatthewPiatetsky) [inline credentials/apps/credentials/models.py]: Did you have something in mind?  
I don't think we always put help text unless it is actually helpful

Comment (clintonb) [inline credentials/apps/credentials/models.py]: There doesn't need to be help text. The users of this application know what the field is for. If they don't...they probably shouldn't be using the application. That may seem a bit snarky, but it's true.

**Comment by MatthewPiatetsky** [comment]
thanks, I missed that last one, removed

File: credentials/apps/credentials/models.py (hunk 1 above)
Comment (davec-edx) [inline credentials/apps/credentials/models.py]: We had it on program_id, maybe the same text....maybe just a comment that if they don't know what the field is, maybe they shouldn't use it :-)

Comment (MatthewPiatetsky) [inline credentials/apps/credentials/models.py]: I agree with Clinton, id/uuid fields should be self explanatory (and the help text we currently had just said that the old field was deprecated)

**Pull request merged**